src/test/java/study/StringTest.java

```diff
@@ -1,13 +1,90 @@
 package study;
 
+import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.*;
 
 public class StringTest {
     @Test
     void replace() {
+        //given
+        //when
         String actual = "abc".replace("b", "d");
+
+        //then
         assertThat(actual).isEqualTo("adc");
     }
+    @Test
+    @DisplayName("요구사항1: \"1,2\"을 ,로 split 했을 때 1과 2로 잘 분리되는지 확인하는 학습 테스트를 구현한다.")
+    void split_test_expect_string_array() {
+        //given
+        String targetString = "1,2";
+
+        //when
+        String[] actual = targetString.split(",");
+
+        //then
+        assertThat(actual).contains("1");
+        assertThat(actual).contains("2");
+        assertThat(actual).containsExactly("1", "2");
```

Comment:
저도 이번에 알았는데 assertThat 메소드가 **체이닝이 가능**한 것 같네요 :)
어떤게 가독성이 더 있는진 모르겠지만 일단 방법은 공유드립니다~

```suggestion
        assertThat(actual)
            .contains("1")
            .contains("2")
            .containsExactly("1", "2");
```